### DISCUSSION TITLE
Prisma create similar table from many-to-many relation with _OrignalName

### DISCUSSION THREAD

**Discussion by mxcdh** [prisma/prisma#18466] [Q&A]
Hello, 

I don't know why after `npx prima migrate dev`, 
prisma creates a similar table to PaginationToProduct, but with _PaginationToProduct name. 

This table is necessary? I want only one table PaginationToProduct, without _PaginationToProduct.

Bellow my schema.prisma and table graph:
https://dsh.re/7f29be
https://dsh.re/c0937

**Comment by ludralph** [comment]
Hi @mxcdh 👋 

`_PaginationToProduct` table is automatically generated for the relation using the name field which you provided in `@@unique([productId, paginationId], name: "PaginationToProduct")`.  it is all managed by Prisma and you do not have to worry about it.

**Reply by mxcdh** [reply]
@ludralph  ok, but is empty. This is necessary? I don't use this table.

**Reply by mxcdh** [reply]
@ludralph  please look, I added to this table another relation, 
and prisma don't create similar auto table like _PaginationToProduct. Why?

https://dsh.re/91297
https://dsh.re/c9e3d

**Reply by ludralph** [reply]
Hi @mxcdh, Apologies for the late reply here. I have taken a good look at the relations you defined, and i observed you are using implicit many-to-many relations between `Product`and `Pagination`. Based on this information, the relation table will be automatically created for you by Prisma. Look at this [section](https://www.prisma.io/docs/concepts/components/prisma-schema/relations/many-to-many-relations#implicit-many-to-many-relations:~:text=Implicit%20many%2Dto%2Dmany%20relations%20define%20relation%20fields%20as%20lists%20on%20both%20sides%20of%20the%20relation.%20Although%20the%20relation%20table%20exists%20in%20the%20underlying%20database%2C%20it%20is%20managed%20by%20Prisma%20and%20does%20not%20manifest%20in%20the%20Prisma%20schema.) of the docs for more information.

**Reply by ludralph** [reply]
Hi @mxcdh 

Do you have any more questions around this issue? If no, it would be really helpful if you could quickly **mark this Discussion as _answered_** to indicate that your issue has been resolved. 

In case you do have any outstanding questions, feel free to let us know and we'd be happy to help!
